packages/components/src/public/QueryColumn.ts

```diff
@@ -232,6 +232,7 @@ export class QueryColumn extends Record({
     static DATA_INPUTS = 'DataInputs';
     static MATERIAL_INPUTS = 'MaterialInputs';
     static ALIQUOTED_FROM = 'AliquotedFrom';
+    static ALIQUOTED_FROM_CAPTION = 'Aliquoted From';
```

Comment:
can we use this constant in more places? e.g. [SampleAliquotDetailHeader.tsx](https://github.com/LabKey/labkey-ui-components/pull/996/files#diff-a1b2dbe896a5385e45a5bc88425ef6e014071dc50ac4e0469690e18a7ab28a3f). Or make one of those constant dictionaries like we do with Schema names?